itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosExclusiveReplica.java

```diff
@@ -74,6 +75,28 @@ public void tearDown() throws Throwable {
     super.tearDown();
   }
 
+  @Test
+  public void testReplOperationsNotCapturedInNotificationLog() throws Throwable {
+    List<String> withClauseOptions = getStagingLocationConfig(replica.repldDir, true);
+    withClauseOptions.add("'" + HiveConf.ConfVars.HIVE_IN_TEST_REPL.varname + "'='" + false + "'");
+    //Perform empty bootstrap dump and load
+    primary.dump(primaryDbName, withClauseOptions);
+    replica.load(replicatedDbName, primaryDbName, withClauseOptions);
+    //Perform empty incremental dump and load so that all db level properties are altered.
+    primary.dump(primaryDbName, withClauseOptions);
+    replica.load(replicatedDbName, primaryDbName, withClauseOptions);
+
+    long lastEventId = primary.getCurrentNotificationEventId().getEventId();
+    primary.dump(primaryDbName, withClauseOptions);
+    long currentEventId = primary.getCurrentNotificationEventId().getEventId();
```

Comment:
Probe that there is no event dir created?

Reply:
Done.